python/include/PythonUnary.h

```diff
@@ -15,33 +18,33 @@ namespace lightdb::python {
                   callable_(callable)
             { 
                 CHECK_NOTNULL(callable);
-                callable->ob_refcnt++; 
+                callable->ob_refcnt++;
```

Comment:
Isn't there a function that should be called here to do this?  Remove dead code below.